keep/alertmanager/alertmanager.py

```diff
@@ -87,4 +87,5 @@ def _run_alerts(self, alerts: typing.List[Alert]):
                 self.logger.error(
                     f"Error running alert {alert.alert_id}", extra={"exception": e}
                 )
+                raise
```

Comment:
so if 1 fails we fail all the others as well?

Reply:
Removed and lets see how things goes (probably should be attribute for `alerts` in the yaml)